src/ert/simulator/simulation_context.py

```diff
@@ -153,25 +153,25 @@ def isRunning(self) -> bool:
         return self._sim_thread.is_alive() or self._job_queue.is_active()
 
     def getNumPending(self) -> int:
-        return self._job_queue.count_status(JobStatus.PENDING)  # type: ignore
+        return self._job_queue.count_status(RealizationState.PENDING)  # type: ignore
 
     def getNumRunning(self) -> int:
-        return self._job_queue.count_status(JobStatus.RUNNING)  # type: ignore
```

Comment:
I think these five functions are only u sed in `batch_simulator_context()` line 55-60. We can get rid of these functions and rather provide these statistics directly from job_queue.

Reply:
done